docs/dev.md

```diff
@@ -28,6 +28,10 @@ The following env variables must be set in order to deploy dev cluster on AWS:
 - `AWS_ACCESS_KEY_ID`
 - `AWS_SECRET_ACCESS_KEY`
 
+The following env variables are recommended although not neccessary for deployment.
```

Comment:
Why it's recommended? 
I suggest using word "optional" and also don't forget to add `AWS_REGION` here and mention that it has default value

Reply:
Done. Changed it to optional and added a small explanation too.

Reply:
When I said default, I meant the one that you actually set in the Makefile. 

Why you're mentioning `~/.aws/config`? We're using `envsubst` it doesn't know about `~/.aws/config`.

Reply:
Done.